app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -305,7 +324,6 @@ class VaultRepositoryImpl(
 
     private fun clearUnlockedData() {
         mutableCiphersStateFlow.update { DataState.Loading }
-        mutableDomainsStateFlow.update { DataState.Loading }
```

Comment:
The domain data should not be tied to vault state (it is not encrypted), but it is still cleared when the user changes, you can see that [here](https://github.com/bitwarden/android/pull/4248/files#diff-36a53931325b5f4d31ddab85e592e43386a56f761f25f7280340a7f019128fd1R231).